src/main/java/uk/gov/di/ipv/cri/common/library/error/OauthErrorResponse.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.di.ipv.cri.common.library.error;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonFormat(shape = JsonFormat.Shape.OBJECT)
+public enum OauthErrorResponse {
```

Comment:
Amendments to the following files are required to increment the version number and explain the changes:
- https://github.com/alphagov/di-ipv-cri-lib/blob/LIME-147/RELEASE_NOTES.md
- https://github.com/alphagov/di-ipv-cri-lib/blob/cc1d6490b0e7166786f43ed6a3029e358ba7ca61/build.gradle#L10